contents/docs/feature-flags/common-questions.mdx

```diff
@@ -1,90 +1,178 @@
-With multiple feature flags, users can fall under different points of rollout percentages per flag, so for a "50% rollout" for feature flag A and feature flag B, you won't always have the same users that see both features at once.
+It's possible to disable sending `Feature flag called` event and thus reduce your feature flag costs. To do this, set `send_events = false` when calling `isFeatureEnabled()` or `getFeatureFlag()`. The code below shows how to do this in our various SDKs:
 
----
+<MultiLanguage>
+
+```js-web
+posthog.getFeatureFlag('flag-key',{
+    send_event: false
+})
+```
+
+```node
+await posthog.getFeatureFlag('flag-key',{
+    send_event: false
+})
+```
+
+```python
+enabled_variant = posthog.get_feature_flag('flag-key', 
+    'distinct_id_of_your_user',
+    send_feature_flag_events=False
+    )
+```
+
+```php
+$isMyFlagEnabledForUser = PostHog::isFeatureEnabled(
+    'flag-key', 
+    'distinct_id_of_your_user',
+    [], // Optional. $groups
+    [], // Optional. $personProperties
+    [], // Optional. $groupProperties
+    false, // Optional. $onlyEvaluateLocally
+    false // Required. Set $sendFeatureFlagEvents to false to disable sending "feature flag called" events
+);
+```
+
+```ruby
+enabled_variant = posthog.get_feature_flag(
+    'flag-key', 
+    'distinct_id_of_your_user',
+    send_feature_flag_events: false
+)
+```
+
+```go
+enabledVariant, err := client.GetFeatureFlag(
+            FeatureFlagPayload{
+                Key:        "flag-key",
+                DistinctId: "distinct_id_of_your_user",
+                SendFeatureFlagEvents: BoolPtr(false),
+            })
+
+if enabledVariant == "variant-key" { // replace 'variant-key' with the key of your variant
+    // Do something differently for this user
+}
+```
+
+```react-native
+posthog.getFeatureFlag('flag-key',{
+    send_event: false
+})
+```
+
+```ios
+posthog.getFeatureFlag('my-flag', options:["send_event": false])
+```
+
+```android
+PostHog.with(this).getFeatureFlag(
+    "flag-key",
+    null, // Optional. Default value
+    new HashMap<String, Object>() {{
+        put("send_event", true);
+}});
+```
+
+</MultiLanguage>
+
+
+**Note:** This has a few consequences:
+
+1. The feature flag's usage tab in PostHog will stop showing events, since we can't track them anymore.
+2. [Experiments](/docs/experiments) that depend on [trend goals](/docs/experiments/creating-an-experiment#experiment-goal) won't work since we use this event to calculate relative exposures Convert your trend goals to funnel goals instead.
+
+
+## Why do feature flags sometimes cause my app to flicker?
+
+By default, fetching flags from our servers takes about 100-500ms. During this time, the flag is disabled. When the request is complete, the flag is updated. This may be the cause of the flickering.
```

Comment:
```suggestion
By default, evaluating flags from our servers takes about 100-500ms. During this time, the flag is disabled. When the request is complete, the flag is updated. This may be the cause of the flickering.
```